com.unity.shadergraph/Editor/GraphUI/GraphElements/Stylesheets/StaticPortParts/ColorPart.uss

```diff
@@ -0,0 +1,3 @@
+.unity-color-field__input {
+    width: 60px;
```

Comment:
Can we make the color picker half as wide as shown in your screen shot for the PR? I know it's smaller than it was before, but it still feels too long to me.

Reply:
Updated the screenshots. It's a bit more than half since the title is pushing out the length of the node, but I shaved some pixels off the preview to make up for it. Do you think this will do until we get to a proper design pass?